schema.sql

```diff
@@ -22,3 +22,14 @@ postgres=# \d vet_clinic
 -- Add a column species of type string to your animals table. Modify your schema.sql file.
 postgres=# ALTER TABLE animals
 postgres-# ADD species varchar(100);
+
+
+-- Third task
+
+postgres=# CREATE TABLE owners (id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY, full_name VARCHAR(120), age INT);
+
+postgres=# CREATE TABLE species (id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY, name VARCHAR(120));
+
+postgres=# ALTER TABLE animals DROP COLUMN species;
+
+postgres=# ALTER TABLE animals ADD COLUMN species_id INT,ADD COLUMN owner_id INT,ADD FOREIGN KEY (species_id) REFERENCES species(id), ADD FOREIGN KEY (owner_id) REFERENCES owners(id);
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly add screenshots with the results in your Pull Request description and not in `thirdimage` as you did. i.e after you run the queries you should add their screenshots containing the result in your PR description.
![screenshots](https://user-images.githubusercontent.com/79058364/184044977-af4c9463-b782-4cfe-a8b0-b4e8259e563f.png)

Reply:
Okay, thank you.